lib/dradis/plugins/saint/field_processor.rb

```diff
@@ -0,0 +1,28 @@
+module Dradis
+  module Plugins
+    module Saint
+
+      class FieldProcessor < Dradis::Plugins::Upload::FieldProcessor
+        def post_initialize(args={})
+          @saint_object =
+            "::Saint::#{data.name.capitalize}".constantize.new(data)
```

Comment:
it seems that we can open ourselves to vulnerabilities by taking data from the uploaded file and then using it inside a `#constantize` call.

If this is the best way to implement this piece, I'd add a line before this one that matched `data.name.capitalize` against a whitelist of known-good class names.